Automattic/jetpack
# Connection Banners: Add premium services to last slide

Connection Banners: Add premium services to last slide.

Currently the last slide of the connection banner doesn't really offer much valuable information. We're enhancing it to talk about our premium services. 



Fixes: https://github.com/Automattic/jetpack/issues/10683

#### Changes proposed in this Pull Request:



BEFORE:
![screen shot 2018-12-06 at 7 25 43 pm](https://user-images.githubusercontent.com/214813/49620311-57e38180-f98e-11e8-85de-87f236a1ebc3.png)



AFTER:
![screen shot 2018-12-08 at 3 20 29 pm](https://user-images.githubusercontent.com/214813/49690337-2f2dca00-fafd-11e8-8d07-3dcd9f7a4aaa.png)


#### Testing instructions:
1) Load this branch. Disconnect jetpack on your site. 
2) View `wp-admin/index.php` or `/wp-admin/plugins.php`
3) Navigate to the last slide within the connection banner. It should say "Premium Services" on the tab and the content in the AFTER screenshot above should load

**Content:**

> 
> *All the tools you’ll ever need for a more powerful WordPress site*
> 
> From hobbyists to large organizations — Jetpack Premium Services keep your site safer, faster, and help you earn more.
> 
> * Real-time automated backups & malware scanning: Stay free of any hacking-related worriess.
> * Ad network: Automatic approval to display ads that generate income from your traffic.
> * Simple payment buttons: Sell products on your site and accept instant PayPal payments.
> * Enhanced search: A WordPress search tool that gives readers faster and more relevant results.
> * Priority support & concierge setup: A one-on-one session to get the most out of your Jetpack site, and priority access to our WordPress experts.
> 
> Set up Jetpack to get started or visit our site to learn more about	all of our features.

#### Proposed changelog entry for your changes:

No changelog entry for this enhancement as it does not affect the direct Jetpack experience for current customers. Just the marketing for potential customer who have not connected to wordpress.com yet.
